generator/generator.go

```diff
@@ -223,6 +236,13 @@ func (g *Generator) Generate(f *ast.File) ([]byte, error) {
 		if err != nil {
 			return vBuff.Bytes(), errors.WithMessage(err, fmt.Sprintf("Failed writing enum data for enum: %q", name))
 		}
+
+		for _, t := range g.userTemplates {
```

Comment:
Also, we should execute these in a known order... that can either by alphabetical by template name, or we can store them in an array instead of a map so that after parsing them, we'd know that they were in the order specified on the command line.